scripts/local-clusters/profiles/single-node-cache.yaml

```
@@ -0,0 +1,42 @@
+# all-in-one cluster with cache
+apiVersion: kind.x-k8s.io/v1alpha4
+kind: Cluster
+containerdConfigPatches:
+  - |-
+    [plugins."io.containerd.grpc.v1.cri".registry]
+      config_path = "/etc/containerd/certs.d"
+networking:
+  disableDefaultCNI: true
+  ipFamily: ipv4
+  podSubnet: 10.244.0.0/16
+  serviceSubnet: 10.96.0.0/12
+nodes:
+  - role: control-plane
+    extraMounts:
+      - containerPath: /etc/containerd/certs.d
```

Couldn't this be `conf.d`? That's where capi puts it https://github.com/elastisys/ck8s-cluster-api/blob/main/helmfile.d/upstream/openstack-cluster/templates/_helpers.tpl#L94C27-L94C31
```suggestion
      - containerPath: /etc/containerd/conf.d
```
But maybe this is not the same

---

Yeah, this is [the recommended way to configure it](https://github.com/containerd/containerd/blob/main/docs/cri/config.md#registry-configuration) and the way we do it in the Cluster API repo [is deprecated](https://github.com/containerd/containerd/blob/main/docs/cri/registry.md).

I think if we have it as `conf.d` it would conflict with the configuration of containerd itself.

---

Aah, I see. Yea, then lets do it this way :+1: